salt/utils/napalm.py

```diff
@@ -31,6 +31,7 @@
     # pylint: disable=unused-import,no-name-in-module
     import napalm
     import napalm.base as napalm_base
+    from napalm.base.exceptions import ConnectionClosedException
```

Comment:
shouldn' t lines 33 and 34 be getting deleted not added if removing napalm base ?

Reply:
@dmurphy18 `napalm-base` was a separate package/library that we were importing.  This one is part of the main napalm library.